src/Controls/src/Core.Design/KnownValuesDesignTypeConverter.cs

```diff
@@ -13,13 +13,13 @@ public abstract class KnownValuesDesignTypeConverter : TypeConverter
 		// This tells XAML this converter can be used to process strings
 		// Without this the values won't show up as hints
 		public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
-			=> sourceType == typeof(string) || base.CanConvertFrom(context, sourceType);
+			=> sourceType.FullName == typeof(string).FullName || base.CanConvertFrom(context, sourceType);
```

Comment:
How come sourceType == typeof(string) doesn't work? That's what we do in our code.

Reply:
Was grasping at straws. It seems fine and I can likely revert this to do the same.